Replaced the BSD-style variable name PREFIX in the makefile with DESTDIR.
--- a/Makefile
+++ b/Makefile
@@ -22,10 +22,10 @@
 #OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE 
 #OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-PREFIX?=/usr/local
-INSTALLBIN=$(PREFIX)/bin
-INSTALLSHARE=$(PREFIX)/share/games
-INSTALLMAN=$(PREFIX)/share/man
+DESTDIR?=/usr/local
+INSTALLBIN=$(DESTDIR)/games
+INSTALLSHARE=$(DESTDIR)/share/games
+INSTALLMAN=$(DESTDIR)/share/man
 
 CC?=gcc
 AR?=ar
